amazon-kinesis-client-multilang/src/test/java/software/amazon/kinesis/multilang/MultiLangDaemonConfigTest.java

```diff
@@ -35,62 +31,168 @@
 import software.amazon.awssdk.auth.credentials.AwsCredentials;
 import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider;
 import software.amazon.kinesis.multilang.config.KinesisClientLibConfigurator;
-import software.amazon.kinesis.multilang.config.MultiLangDaemonConfiguration;
 
 @RunWith(MockitoJUnitRunner.class)
 public class MultiLangDaemonConfigTest {
-    private static String FILENAME = "some.properties";
+    private static final String TEST_FILENAME = "some.properties";
+    private static final String TEST_EXE = "TestExe.exe";
+    private static final String TEST_APPLICATION_NAME = "TestApp";
```

Comment:
FYI: rather than a generic literal like "TestApp", I often use something like `MLDCTest.class.getSimpleName()` Subjectively, this helps clarify any output/logs.